modules/doc/content/ncrc/ncrc_local_binary.md

```diff
@@ -42,144 +42,176 @@ optional arguments:
-  Ran 1606 tests in 274.6 seconds. Average test time 0.7 seconds, maximum test time 20.7 seconds.
-  1606 passed, 22 skipped, 0 pending, 0 failed
-  ```
-
-- Run a specific input file
-
-  ```bash
-  $> conda activate bison
-  $ (bison) >
-  $ (bison) > cd ~/path/to/my/input/file
-  $ (bison) > bison-opt -i my_input.i
-
-  # Using MPI
-  $ (bison) > mpiexec -n 8 bison-opt -i my_input.i
-  ```
-
-- View Documentation
-
-  ```bash
-  $> conda activate bison
-  $ (bison) >
-  $ (bison) > echo $bison_DOCS
-  /home/johndoe/miniconda3/envs/bison/bison/share/bison/doc/index.html
-  ```
-
-  You can then enter that location into your favorite web browser. Similarly on Macintosh machines, you can make use of the `open` command, to open the environment variable:
-
-  ```bash
-  $> conda activate bison
-  $ (bison) > open $bison_DOCS
-  ```
+Where applicable, you only need to activate the environment once. Each example below assume the environment has not yet been activated.
+
+##### Running Tests
+
+To run the application's tests, you first instruct that application to copy its test files into a usable directory, enter said directory, and then run the tests. As follows:
+
+```bash
+$> conda activate bison
+$ (bison) >
+$ (bison) > bison-opt --copy-inputs tests
+
+<Trimmed results. Output consists of tests being copied.>
+
+$ (bison) > cd bison/tests
+$ (bison) > bison-opt --run -j 5
+fipd_rodlet_mesh_generator.test ........................................................................... OK
+fill_gas_thermal_conductivity.legacy.He ................................................................... OK
+<trimmed>
+triso_pebble.3d_pebble_from_csv .............................................................. [min_cpus=5] OK
+triso_pebble.cylinder_2d_rz ............................................................................... OK
+triso_pebble.cylinder_3d .................................................................................. OK
+triso_pebble.sphere_2d_rz ................................................................................. OK
+triso_pebble.sphere_3d .................................................................................... OK
+triso_pebble.plate ........................................................................................ OK
+--------------------------------------------------------------------------------------------------------------
+Ran 1606 tests in 274.6 seconds. Average test time 0.7 seconds, maximum test time 20.7 seconds.
+1606 passed, 22 skipped, 0 pending, 0 failed
+```
+
+##### Some Applications contain more than tests
+
+Some applications have different inputs that can be copied and executed. Each application can use whatever naming convention
+they wish, so listing all possibilities is impossible. However, any runnable input directory will have a file called `testroot`
+residing in it. The following *crazy* command will return those possibilities (using Bison as an example):
```

Comment:
🤪

Reply:
yeah... hahahaha any help here in making this... better described is appreciated!

Reply:
Added another comment with a wording suggestion that you can take or leave. I appreciate that you are giving a resource for the user to find what those input categories actually are.

Reply:
Yeah, I then need to add what to actually do with these. Basically another example of how one runs tests.